ksql-engine/src/test/java/io/confluent/ksql/ddl/commands/CreateSourceCommandTest.java

```diff
@@ -146,12 +181,61 @@ public void shouldNotThrowIfTopicDoesExist() {
     when(kafkaTopicClient.isTopicExists(TOPIC_NAME)).thenReturn(true);
 
     // When:
-    new TestCmd("what, no value topic?", statement, ksqlConfig, kafkaTopicClient, serdeOptions);
+    new TestCmd(
+        "what, no value topic?",
+        statement,
+        ksqlConfig,
+        kafkaTopicClient,
+        serdeOptions,
+        serdeFactories
+    );
 
     // Then:
     verify(kafkaTopicClient).isTopicExists(TOPIC_NAME);
   }
 
+  @Test
+  public void shouldThrowIfTopicToCreateDoesExist() {
```

Comment:
Can we rename to differentiate from tests around the actual topic existing in Kafka or not?

```suggestion
  public void shouldThrowIfTopicWithSameNameAlreadyRegistered() {
```

Reply:
Done